### PR TITLE
Create sly.json

### PR DESCRIPTION
**Pull request by SlyBouhafs**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

 https://slybouhafs.github.io

image

### CODE REVIEW

File: domains/sly.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Domain name for my personal portfolio website",
+    "repo": "https://github.com/SlyBouhafs/slybouhafs.github.io",
+    "owner": {
+        "username": "SlyBouhafs",
+        "email": "khmedj+github@gmail.com",
```
Comment (CuteDog5695) [inline domains/sly.json:6]:
```suggestion
        "email": "khmedj+github@gmail.com"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by MaskDuck — DISMISSED** [review]
oh my fucking god you use nvim for coding fuiyoh maskduck take my hat off to you omg omg omg

you have not been brainwashed by stupid corporates like Windows who lure you into using VSCode, that's a really good thing!

I love you x 1000

image

File: domains/sly.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "username": "SlyBouhafs",
+        "email": "khmedj+github@gmail.com"
+    },
+    "record": {
+        "CNAME": "slybouhafs.github.io",
+        "MX": [
+              "mx1.improvmx.com",
+              "mx2.improvmx.com"
+        ],
+        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```
Comment (CuteDog5695) [inline domains/sly.json:14]: You cannot use CNAME with MX and TXT.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/sly.json (hunk 2 above)
Comment (SlyBouhafs) [reply domains/sly.json:14]: Hey, thanks for the feedback!
How do i use it? Do you have to create another file for it?

**Pull request closed without merging**